dohnut.js

```diff
@@ -25,6 +43,23 @@ const { origin, pathname } = new URL(doh)
 const client = http2.connect(origin)
 const socket = dgram.createSocket('udp4')
 
+const dumb = (id, rec) => {
+  return dnsPacket.encode({
+    type: 'response',
```

Comment:
https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml

https://github.com/mafintosh/dns-packet/blob/0f87ff09f523447165848086534c477ca51337fd/rcodes.js

Refused? NXdomain?

Reply:
I opt for lying and replying `NXDOMAIN`

Reply:
SGTM